cmd/go-filecoin/daemon.go

```diff
@@ -163,7 +164,10 @@ func RunAPIAndWait(ctx context.Context, nd *node.Node, config *config.APIConfig,
 		inspectorAPI:   NewInspectorAPI(nd.Repo),
 		porcelainAPI:   nd.PorcelainAPI,
 		//retrievalAPI:   nd.RetrievalProtocol.RetrievalClient,
-		//storageAPI:     nd.StorageProtocol.StorageClient,
+	}
+
+	if nd.StorageProtocol != nil {
```

Comment:
Nodes don't necessarily have storage protocols in test environments. Add this only if it does.